src/main/java/com/scriptbasic/executors/operators/AbstractBinaryFullCircuitOperator.java

```diff
@@ -15,14 +15,38 @@
  */
 public abstract class AbstractBinaryFullCircuitOperator extends
         AbstractBinaryOperator {
+    
+    /**
+     * Function used to evaluate left operand.
+     * Override this function if custom evaluation is needed.
+     * 
+     * @param interpreter 
```

Comment:
I know that this is stupid because the name says it all, but modern JavaDoc compiler complains if there is no description. In this case, I put there a `.` nowadays.